README.txt

```diff
@@ -0,0 +1,32 @@
+= pmdtester
```

Comment:
I'd rename this file to `README.md`, as it should be a Markdown file rather than a simple text file

Reply:
Done.